Mods/ArchipelagoPlayer.SC2Mod/Preload.xml

```diff
@@ -7,7 +7,6 @@
 <Actor id="AP_SOAGravitonBombTargetSound"/>
 <Actor id="AP_SOAOrbitalStrikeTargetModel"/>
 <Actor id="AP_SOATargetingModeConfirmationSound"/>
-<Actor id="AP_Stargate"/>

```

Comment:
Curious if the editor removed this or if you did.

Reply:
I remember seeing this when resolving merge conflicts, I must have done something wrong when fixing conflicts, but I can't remember what.  It did stick out to me as odd though.